README.md

```diff
@@ -36,12 +36,29 @@ mailing list.
 
 ## Installation
 
+### Bazel
+
+If you build with `bazel`, follow the [`bazel` documenation for referencing
+external projects][bazel-external-deps] to include Dagger in your build.
+
+Given the following `WORKSPACE` definition, you can reference dagger via
+`@com_google_dagger//:dagger-with-compiler` in your deps.
```

Comment:
It's actually called `dagger_with_compiler`, with underscores rather than hyphens.

You may want to add a note about producers, grpc, Android et al. Or are they all provided by this dependency?